tests/DfE.FindInformationAcademiesTrusts.UnitTests/Pages/Trusts/Academies/AcademiesPageModelTests.cs

```diff
@@ -112,4 +122,25 @@ public async Task OnGetAsync_should_configure_TrustPageMetadata()
         _sut.TrustPageMetadata.PageName.Should().Be("Academies");
         _sut.TrustPageMetadata.TrustName.Should().Be("My Trust");
     }
+
+    [Fact]
+    public async Task OnGetAsync_sets_correct_data_source_list()
+    {
+        TrustSummaryServiceModel fakeTrust = new("1234", "My Trust", "Multi-academy trust", 3);
+        _mockTrustService.Setup(t => t.GetTrustSummaryAsync(fakeTrust.Uid)).ReturnsAsync(fakeTrust);
+        _sut.Uid = fakeTrust.Uid;
+        
+        _ = await _sut.OnGetAsync();
+        _mockDataSourceService.Verify(e => e.GetAsync(Source.Gias), Times.Once);
+        _mockDataSourceService.Verify(e => e.GetAsync(Source.ExploreEducationStatistics), Times.Once);
+        _sut.DataSourcesPerPage.Should().BeEquivalentTo([
+            new DataSourcePageListEntry("Details", [new DataSourceListEntry(_giasDataSource)]),
```

Comment:
Consider a central source of these regularly used strings, they are all used quite a few times we should have consts